Dockerfiles/midstream/README.md

```diff
@@ -15,18 +15,18 @@ First thing you reckon is that you are running this command directly from upstre
 
 ```
 git clone https://github.com/redhat-operator-ecosystem/operator-test-playbooks.git
-cd Dockerfiles
+cd operator-test-playbooks
 ### podman build -t <image_name>:<tagname> -f <filepath> --build-arg OPERATOR_SDK_VERSION=<operator_sdk_version> --build-arg OPERATOR_TEST_PLAYBOOKS_TAG=<operator_test_playbooks_tag>
 ### example:
-podman build -t midstream_image:latest -f Dockerfile --build-arg OPERATOR_SDK_VERSION=v1.4.0 --build-arg OPERATOR_TEST_PLAYBOOKS_TAG=v1.0.11
+podman build -t midstream_image:latest -f Dockerfile . --build-arg OPERATOR_SDK_VERSION=v1.4.0 --build-arg OPERATOR_TEST_PLAYBOOKS_TAG=master
```

Comment:
OPERATOR_TEST_PLAYBOOKS_TAG does not exist anymore

Reply:
and we shoudl rely in default for `OPERATOR_SDK_VERSION`

Reply:
removed